lua/modules/chat.lua

```diff
@@ -219,10 +219,13 @@ local createModalContent = function(_, config)
 
 		-- check if command
 		local channelCommand = string.sub(message, 1, 3)
-		if channelCommand == "/a " or channelCommand == "/l " or channelCommand == "/w " then
```

Comment:
why did we even check for `a`, `l` and `w`?

Reply:
Its for choosing who will receive the message, respectively all, local and whisper